### DISCUSSION TITLE
Help with Custom Subdomains

### DISCUSSION THREAD

**Discussion by MiragianCycle** [community/community#23679] [Other Feature Feedback, Questions, & Ideas]
<p>Hello.</p>
<p>My GitHub page is set to the custom domain <a href="http://theena.net" rel="noopener nofollow ugc">theena.net</a>. This is working well and I have no problems here.</p>
<p>What I’d like to do now is to add a sub-domain. I’d like this subdomain to redirect to a separate repo (gh/MiragianCycle/MiragianStudios)</p>
<p>The support team at my domain provider (namecheap) helped ensure that I’d set my DNS records right. Now I am faced with a 404 error when going to the subdomain. This is strange cause i have an index.html for this repo.</p>
<p>Apologies if this has been asked to death but I didn’t see anything that matched my particular case which is why I am starting a new thread.</p>
<p>Thanks in advance.</p>

**Comment by MiragianCycle** [comment]
<p>The system isn’t letting me add more than 2 links so here is the sub domain in question: <a href="http://miragianstudios.theena.net/" rel="noopener nofollow ugc">http://miragianstudios.theena.net/</a></p>

**Comment by Maximization** [comment, marked as answer]
<p>If I query the DNS records of <a href="http://miragianstudios.theena.net" rel="noopener nofollow ugc">miragianstudios.theena.net</a>, I can’t seem to find any A or CNAME type records. How does your configuration look like in namecheap?</p>

**Comment by otter-computer** [comment]
<p>Looks like <span class="mention">@Maximizationis</span> right, there isn’t a <code>CNAME</code> DNS record set for that subdomain to point it towards GitHub.</p>
<p><a href="https://www.whatsmydns.net/#CNAME/miragianstudios.theena.net">https://www.whatsmydns.net/#CNAME/miragianstudios.theena.net</a></p>
<p>You’ll need to create a new <code>CNAME</code> record for the <code>miragianstudios</code> subdomain that points towards your GitHub address <code>MiragianCycle.github.io</code>.</p>
<p>There’s some more detailed instructions on how to do that here:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain" target="_blank" rel="noopener">Managing a custom domain for your GitHub Pages site - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by MiragianCycle** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/6/26f429be2a7f4a2ab8bf3baf5fd177f73847f178.png" data-download-href="/uploads/short-url/5yBgazMTQR8mskjBLGyDurc9H72.png?dl=1" title="today" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/58807314/181078728-f43f3556-4b06-417a-9d01-4ad76d44d186.png" alt="today" data-base62-sha1="5yBgazMTQR8mskjBLGyDurc9H72" width="690" height="259" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/6/26f429be2a7f4a2ab8bf3baf5fd177f73847f178_2_690x259.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/6/26f429be2a7f4a2ab8bf3baf5fd177f73847f178_2_1035x388.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/6/26f429be2a7f4a2ab8bf3baf5fd177f73847f178.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/6/26f429be2a7f4a2ab8bf3baf5fd177f73847f178_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">today</span><span class="informations">1248×469 24.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Did that. Now DNS is back to not being able to find the server. Here’s a screenshot of my Advance DNS tab on namecheap.</p>

**Comment by MiragianCycle** [comment]
<p>Problem solved. Thank you both for your help.</p>
